crates/extensions/c8y_mapper_ext/src/config.rs

```diff
@@ -111,6 +111,12 @@ impl C8yMapperConfig {
             "tedge/events/+/+",
             "tedge/health/+",
             "tedge/health/+/+",
+            "te/device/+///m/",
+            "te/device/+///m/+",
+            "te/device/+///a/",
+            "te/device/+///a/+",
+            "te/device/+///e/",
+            "te/device/+///e/+",
```

Comment:
As I wrote in the comment in `crates/common/tedge_config/src/tedge_config_cli/new.rs`, I think it's wrong.
```suggestion
            "te/device/+/+/+/m/",
            "te/device/+/+/+/m/+",
            "te/device/+/+/+/a/+",
            "te/device/+/+/+/e/+",
```

Reply:
fixed